ext/zlibpng.cmd

```diff
@@ -5,4 +5,5 @@
 : # The odd choice of comment style in this file is to try to share this script between *nix and win32.
 
 git clone -b v1.3.1 --depth 1 https://github.com/madler/zlib.git
-git clone -b v1.6.40 --depth 1 https://github.com/glennrp/libpng.git
+git apply --ignore-whitespace
```

Comment:
This is missing the patch file name, so the command hangs (presumably waiting for the patch file from stdin).

Fixed in https://github.com/AOMediaCodec/libavif/pull/2743.